.pre-commit-config.yaml

```diff
@@ -1,21 +1,21 @@
 ---
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.6.0
-    hooks:
-      - id: check-added-large-files
-      - id: check-ast
-      - id: check-case-conflict
-      - id: check-merge-conflict
-      - id: check-toml
-      - id: debug-statements
-      - id: detect-private-key
-      - id: end-of-file-fixer
-      - id: forbid-new-submodules
-      - id: trailing-whitespace
+    - repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v5.0.0
+      hooks:
+          - id: check-added-large-files
+          - id: check-ast
+          - id: check-case-conflict
+          - id: check-merge-conflict
+          - id: check-toml
+          - id: debug-statements
+          - id: detect-private-key
+          - id: end-of-file-fixer
+          - id: forbid-new-submodules
+          - id: trailing-whitespace
 
-  - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.2
-    hooks:
-      - id: pyupgrade
-        args: [--py38-plus]
+    - repo: https://github.com/asottile/pyupgrade
+      rev: v3.19.1
+      hooks:
+          - id: pyupgrade
+            args: [--py39-plus]
```

Comment:
If it’s not too much to ask, could you (1) revert the whitespace changes in this file and (2) update the `pyupgrade` version to the most-recent `v3.20.0`?